orb-connd/debian/worldcoin-connd.service

```diff
@@ -12,7 +12,7 @@ WorkingDirectory=/home/worldcoin
 Type=exec
 Environment=DBUS_SESSION_BUS_ADDRESS=unix:path=/tmp/worldcoin_bus_socket
 Environment=RUST_BACKTRACE=1
-ExecStart=/usr/local/bin/orb-connd
+ExecStart=/usr/local/bin/orb-connd connd
```

Comment:
Probably better to just default to the main entry point and make this subcommand the default